updater/fetchers/debian/debian.go

```diff
@@ -67,8 +68,11 @@ func (fetcher *DebianFetcher) FetchUpdate(datastore database.Datastore) (resp up
 	r, err := http.Get(url)
 	if err != nil {
 		log.Errorf("could not download Debian's update: %s", err)
+		io.Copy(ioutil.Discard, r.Body)
```

Comment:
This is only useful when `r.Body` isn't read completely so the TCP connection can be re-used.